python/pyproject.toml

```diff
@@ -22,8 +22,8 @@ dependencies = [
 [project.optional-dependencies]
 srt = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hub", "interegular",
        "packaging", "pillow", "psutil", "pydantic", "python-multipart",
-       "torch", "torchao", "uvicorn", "uvloop", "zmq",
-       "vllm==0.5.5", "outlines>=0.0.44"]
```

Comment:
Why was the `vllm` dependency removed?

Reply:
Thanks for your comments. I have refine the pyproject.toml to add the xpu path. 
 xpu is not enabled in public vllm and torch whl,need to follow https://docs.vllm.ai/en/latest/getting_started/xpu-installation.htmlinstall vllm. The XPU device in pytorch is enabled since torch-2.4, but the vllm with XPU is only support torch-2.3+IPEX now. I will double check with them when it will works with torch-2.4 on XPU device.

Reply:
Please make sure that the NVIDIA GPU functionality remains unaffected while integrating XPU support. For instance, the removal of the vllm dependency in this context has led to unforeseen issues.